rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/CovidPatientRegisterActivity.java

```diff
@@ -18,6 +27,14 @@
  */
 public class CovidPatientRegisterActivity extends PatientRegisterActivity {
 
+    private static final String LOCALE_IN = "in";
```

Comment:
I think this should be in `CovidConstants`?

Reply:
Also, although `in` might work, the new language code for Bahasa is `id`.

Reply:
done.